backend/src/org/akvo/lumen/middleware.clj

```diff
@@ -1,25 +1,26 @@
 (ns org.akvo.lumen.middleware
-  ""
-  (:require
-   [clojure.string :as s]
-   [akvo.commons.jwt :as jwt]
-   [clj-http.client :as client]
-   [ring.util.response :as response]))
+  (:require [akvo.commons.jwt :as jwt]
+            [clj-http.client :as client]
+            [clojure.pprint :refer [pprint]]
```

Comment:
There is a (pprint e) form in wrap-jwt
https://github.com/akvo/akvo-lumen/pull/269/files#diff-eba5c89e38c7b1943887582cee409557R46